test/com/moclojer/server_test.clj

```diff
@@ -62,3 +62,18 @@
                (response-for :get "sub.moclojer.com/multihost-sub")
                :body
                (json/parse-string true))))))
+
+(deftest uri-with-multi-paths
+  (is (= {:hello-v1 "world!"}
+         (-> (helpers/service-fn (yaml/from-file "test/com/moclojer/resources/moclojer.yml"))
+             (response-for :get "/v1/hello/world")
+             :body
+             (json/parse-string true)))))
+
+(deftest uri-with-multi-paths-fixed
+  (is (= {:hello-v1 "hello world!"}
+         (-> (helpers/service-fn (yaml/from-file "test/com/moclojer/resources/moclojer.yml"))
+             (response-for :get "/v1/hello")
+             :body
+             (json/parse-string true)))))
+
```

Comment:
Maybe add a test for a multi path mock with multiple variables in between?

For example a `/users/:username/data/:age`?

Reply:
writer!